src/stk/molecular/molecules/molecule/utilities/updaters/turbomole.py

```diff
@@ -25,16 +26,39 @@ def _with_structure_from_turbomole(self, path):
         number of atoms in the molecule or if atom elements in the
         file do not agree with the atom elements in the molecule.
 
+    :class:`NotImplementedError`
+        If the the Turbomole file has coordinates defined as fractional
+        based on a unit cell.
+
     """
 
     bohr_to_ang = 0.5291772105638411
 
     with open(path, 'r') as f:
-        _, *content, __ = f.readlines()
+        content = f.readlines()
+
+    for i, cont in enumerate(content):
+        if '$coord' in cont:
+            if 'angs' in cont:
+                coord_units = 'angstrom'
+            elif 'frac' in cont:
+                coord_units = 'fractional'
+                raise NotImplementedError(
```

Comment:
Based on https://docs.python.org/3/library/exceptions.html#NotImplementedError I'm not sure this is the right error type. Happy to be convinced otherwise.

Reply:
Done